versioning/scripts/dockerfiles/main.go

```diff
@@ -173,30 +180,39 @@ func check(e error) {
 func main() {
 	templateDirPtr := flag.String("template_dir", "templates", "Path to directory containing Dockerfile.template and any other files to copy over")
 	verifyPtr := flag.Bool("verify_only", false, "Verify dockerfiles")
+	var templatePath string
```

Comment:
not sure what this block is doing?

Reply:
templatePath is to represent a path to Dockerfile.template within a specific subfolder.
templatePath is different from templateDirPtr - the latter points to 'templates' folder, the former points to a subfolder of 'templates'